src/main/resources/META-INF/patches/groovy/4.0.0-migrateSitemap.started.groovy

```diff
@@ -30,13 +31,20 @@ abstract class Scroller extends ScrollableQueryCallback<Void> {
     }
 
     protected void reverseMixins(JCRNodeWrapper node) {
-        if (node.isNodeType("jmix:sitemap")) {
-            node.removeMixin("jmix:sitemap");
-            return;
-        }
 
-        if (node.isNodeType("jnt:page") || node.isNodeType("jmix:mainResource")) {
+        if (node.isNodeType("jmix:sitemap") && node.isNodeType("jmix:noindex")) {
```

Comment:
This seems to be simplified by doing 2 ifs instead of 3 if / else 
if (node.isNodeType("jmix:sitemap")  // Do something
if (node.isNodeType("jmix:noindex") // Do something

Reply:
Yes, but I did it like this to have different logs when a content have the mixin jmix:sitemap AND jmix:noindex as it's asked in the ticket

Reply:
in case of both, I would have put the error log but do nothing, and keep the two ifs that does things.